site/admission/index.html

```diff
@@ -13,7 +13,7 @@
   <nav><a href="">Home</a><a href="/about">About</a></nav>
   <div class="header-box">
     <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle">
-    <h1>Admission</h1>
+    <div class="title">Admission</div>
```

Comment:
I think you could leave this as an h1 and just add the class. So you'd have:

`<h1 class="title">Admission</h1>`